### PR TITLE
Accessibility: Change link color in dark theme

### PR DESCRIPTION
**Pull request by GunnerStraiker1**
## Hey, I just made a Pull Request!

I've changed the link color in dark theme to #EFE5B1 instead of dark blue 

**Before**
Screen Shot 2021-03-24 at 8 56 36

**After**
Screen Shot 2021-03-25 at 14 24 25

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [X] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
before-and-after screenshots would be helpful!

**Comment by GunnerStraiker1** [comment]
@freben Sorry, I forgot it, I've updated the Pull request description

**Comment by freben** [comment]
But that's like yellow - should it not just be a slightly lighter tone of what we had before?

**Comment by GunnerStraiker1** [comment]
@freben Before my change, it was #0A6EBE, like a darker blue, so It's really hard to read on a black background, however, I have also another 2 options A(#B9D5F7), B(#E1C3FD), both are a type of light blue

**Comment by GunnerStraiker1** [comment]
**A(#B9D5F7)**
Screen Shot 2021-03-24 at 10 33 39

**B(#E1C3FD)**
Screen Shot 2021-03-24 at 10 35 19

**Comment by adamdmharvey** [comment]
If it helps, you can check the Backstage Figma design, as it lists the available recommended colors.  I think anything beyond those, at least as it relates to the base out of the box theme, likely should be run by the Backstage Design team for review.  Otherwise, it could be a local themed customization done on a per company basis and not contributed up into the core?

https://www.figma.com/community/file/850673348101741100?preview=fullscreen

image

**Comment by GunnerStraiker1** [comment]
@adamdmharvey Oh, I didn't see that link before, so I'm gonna check it and then I'll add a new change

File: .changeset/light-horses-hammer.md
Comment (adamdmharvey) [inline .changeset/light-horses-hammer.md:5]: Rather than using the personal "I've changed" and putting the color here, this should be more generic. Something more like what you have in the title of the PR: "Change link color in dark theme"?

**Review by adamdmharvey — CHANGES_REQUESTED** [review]

File: .changeset/light-horses-hammer.md
Comment (GunnerStraiker1) [reply .changeset/light-horses-hammer.md]: @adamdmharvey Done

File: .changeset/light-horses-hammer.md
Comment (adamdmharvey) [inline .changeset/light-horses-hammer.md:5]:
```suggestion
Replace the link color in dark theme
```

...suggest you don't need to specify what color it is, since those reviewing the CHANGELOG won't be able to tell what that hex code is anyway.  Instead, if interested they can simply open the change via this PR to see exactly what color you DID choose. Key point for a change record is that you changed the color, and it's for the dark theme, if that makes sense :)

**Review by adamdmharvey — DISMISSED** [review]
Thanks @GunnerStraiker1 !

Can you update the screen grab in the description of the PR with what your new colors look like?

Thinking we may want @katz95 or someone on the design team to give a look, thoughts?

**Comment by GunnerStraiker1** [comment]
@adamdmharvey Done

**Comment by GunnerStraiker1** [comment]
It's ready and waiting for the design team comments/approval, @katz95 @adamdmharvey

**Comment by katz95** [comment]
Hmm I'm a bit wary of using yellow for our link colors, since blue is the go-to. Can we change the values to this?
Screenshot 2021-03-31 at 10 57 46 PM

**Comment by freben** [comment]
Updated after-screengrab after applying @katz95 suggested colors:

Screenshot 2021-04-20 at 19 47 20

**Review by freben — APPROVED** [review]
@GunnerStraiker1 I hope you don't mind I rebased and added the colors that were suggested. 🚢 

I would argue that it's unfortunate that we have these extra link colors; as far as I understand it they should always align with the primary color, so it's just redundant stuff used by very few places.

**Comment by freben** [comment]
@adamdmharvey mind reviewing this again? :)

File: packages/theme/src/themes.ts
Comment (adamdmharvey) [inline packages/theme/src/themes.ts:115]: This color didn't seem to come from @katz95 's suggestion; is this a different one? Apols not sure where this one pops up, or why moving it away from black.

Comment (freben) [reply packages/theme/src/themes.ts]: Ah quite right, i reverted this back since I am honestly not sure what this will look like together with the banner and we should tackle that separately. Done!

**Pull request merged**
